draft-ietf-jsonpath-base.md

```diff
@@ -1272,14 +1272,18 @@ rather than the negated existence test `!@.foo` (which yields false if `@.foo` s
 
 The comparison operators `==` and `<` are defined first and then these are used to define `!=`, `<=`, `>`, and `>=`.
 
-When a path resulting in an empty nodelist appears on either side of a comparison:
+When a path resulting in an empty nodelist or a function expression returning `Nothing` appears on either side of a comparison:
 
 * a comparison using the operator `==` yields true if and only if the comparison
-is between two paths each of which result in an empty nodelist.
+is between:
+    * two paths each of which result in an empty nodelist,
+    * two functions each of which returns `Nothing`, or
+    * a path which results in an empty nodelist and a function which returns `Nothing`.
```

Comment:
If we're listing out cases, we need to add "empty nodelist function == nothing function"

Reply:
Yes. I think I've found a more compact way of writing all this.